src/api/membership.ts

```diff
@@ -49,7 +49,7 @@ export const postItemMembership = async (
   config: PartialQueryConfigForApi,
 ) => {
   const { API_HOST, axios } = config;
-  const member = await getMembersBy({ emails: [email] }, config);
+  const member = await getMembersByEmail({ emails: [email] }, config);
 
   if (!member || !Object.values(member.data).length) {
```

Comment:
maybe it's clearer if we add that as well.
```suggestion
  const members = await getMembersByEmail({ emails: [email] }, config);

  if (!members || !Object.values(members.data).length) {
```